src/app/pages/news/11.3/03-engine-cockpit.md

```diff
@@ -1,6 +1,10 @@
 ## Engine Cockpit {#engineCockpit}
 
-New SSL View, TLS tester...
+The Cockpit includes new features to ease your secure operations.
+
+- **Binaries**: The Config File Editor now also lists binary key- and trust-store files. Using the new up-/download functionality, you can edit them locally with a tool of your preference and put the modified version back in charge later on.
```

Comment:
Grammerly says: "now also" is strange. It suggest to put "now" at the end of the sentence